Plots/MapProjections/NCL_lcnative_1_lg.py

```diff
@@ -0,0 +1,80 @@
+"""
+NCL_polar_1_lg.py
+================
+
+This script illustrates the following concepts:
+    - Drawing contours over a map using a native lat,lon grid
+    - Drawing filled contours over a Lambert Conformal map
+    - Zooming in on a particular area on a Lambert Conformal map
+    - Subsetting a color map
+
+See following URLs to see the reproduced NCL plot & script:
+    - Original NCL script: https://www.ncl.ucar.edu/Applications/Scripts/lcnative_1.ncl
+    - Original NCL plot: https://www.ncl.ucar.edu/Applications/Images/lcnative_1_lg.png
+"""
+
+import numpy as np
+import xarray as xr
+import cartopy.crs as ccrs
+import matplotlib.pyplot as plt
+import geocat.datafiles as gdf
+from geocat.viz import cmaps as gvcmaps
+from geocat.viz import util as gvutil
+import matplotlib.ticker as ticker
```

Comment:
I would move the ticker import next to the plt import. We should group import statements from the same package together and put the geocat imports at the end for clarity. That way, the imports are organized by common packages (matplotlib.pyploy, cartopy) at the top and less common (geocat.util) at the bottom.